crafters/audio/AudioMonophoner/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM jinaai/jina
+FROM jinaai/jina:1.0.15
```

Comment:
Will the jina version be updated automatically when the update action is triggered?

Reply:
Yes, Example PR: https://github.com/jina-ai/jina-hub/pull/7096/